{{cookiecutter.project_name}}/next_steps.md

```diff
@@ -0,0 +1,36 @@
+# Next steps
+
+## Put the generated files under version control
+
+Once your Python package is created, put it under [version
+control](https://guide.esciencecenter.nl/#/best_practices/version_control) using
+[git](http://git-scm.com/) and [GitHub](https://github.com/).
+
+```shell
+cd {{ cookiecutter.project_name }}
+git init
+git add --all
+git commit -m "first commit"
+git branch -M main
+git remote add origin {{ cookiecutter.repository }}
+```
+
+## Push the initial commit to a new repo on GitHub
+
+Go to
+[https://github.com/{{cookiecutter.github_organization}}?tab=repositories](https://github.com/{{cookiecutter.github_organization}}?tab=repositories)
+and create a new repository named {{ cookiecutter.project_name }} as an empty repository, then push your commits to GitHub:
+
+```shell
+git push --set-upstream origin main
+```
+
+## Check automatically generated issues
+
+The first time you push to GitHub, a few issues outlining next steps will be added automatically
```

Comment:
It might takes some time for the GHA workflow to run. Maybe mention it here?
```suggestion
The first time you push to GitHub, a few issues outlining next steps will as soon as possible be added automatically
```